src/pages/transfer/transfer.html

```diff
@@ -2,6 +2,81 @@
     <smilo-nav-header titleTranslationKey="transfer.nav-bar-title"></smilo-nav-header>
 </ion-header>
 
-<ion-content padding>
-
+<ion-content padding *ngIf="fromWallet">
+    <ion-grid>
```

Comment:
Consider using [Angular Forms](https://angular.io/guide/form-validation) to do the validation.

Reply:
Created an issue for another time: https://github.com/Smilo-platform/Smilo-Wallet/issues/174